### PR TITLE
Add ERC: Simple token, Simplified ERC-20

### PR DESCRIPTION
**Pull request by wenzhenxiang**
Migrated from eips https://github.com/ethereum/EIPs/pull/7196

This EIP is a new asset designed based on the user contract wallet (including AA), and is forward compatible with [ERC-20](./eip-20.md).

Ethereum-magicians [discussion](https://ethereum-magicians.org/t/simple-token-designed-for-smart-contract-wallet-aa/14757/1)

### CODE REVIEW

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: ERCS/erc-7196.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+eip: 7196
+title: simple token
+description: Designed for smart contract wallet
+author: Xiang (@wenzhenxiang), Ben77 (@ben2077), Mingshi S. (@newnewsms)
+discussions-to: https://ethereum-magicians.org/t/simple-token-designed-for-smart-contract-wallet-aa/14757
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-21
+---
```
Comment (SamWilsn) [inline ERCS/erc-7196.md:11]:
```suggestion
requires: 20
---
```

Enough of your proposal is based on ERC-20, so requiring it makes sense. Note that this doesn't mean implementers have to implement the `ERC20` interface.

File: ERCS/erc-7196.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+eip: 7196
+title: simple token
+description: Designed for smart contract wallet
```
[the remark this answers is not in the document]
Comment (wenzhenxiang) [reply ERCS/erc-7196.md:4]: Thank you very much, I will make changes based on all your suggestions.

**Comment by wenzhenxiang** [comment]
@SamWilsn Thank you for your valuable suggestions and insights, I've made important revisions based on your suggestions.

**Review by SamWilsn — DISMISSED** [review]

**Pull request merged**